docs/change-log/2025-04-21-components-v2.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Introducing New Components for Messages!"
+date: "2025-04-21"
+topics:
+- "User Apps"
+- "HTTP API"
+- "Interactions"
+---
+
+We're bringing new components to messages that you can use in your apps. They allow you to have full control over the layout of your messages.
+
+#### Why We Built Components V2
+
+Our previous components system, while functional, had limitations:
+
+- Content, attachments, embeds, and components had to follow fixed positioning rules
+- Visual styling options were limited
+
+Our new component system addresses these challenges with fully composable components that can be arranged and laid out in any order, allowing for a more flexible and visually appealing design.
+
+#### What's New
+
+[Components V2](/docs/components/overview) introduces several new component types that can be used in messages:
+
+#### New Layout Components
+
+- [**Section**](/docs/components/reference#section) - Combine text with an accessory component for contextually linked elements
+- [**Container**](/docs/components/reference#container) - Create visually distinct groupings with a customizable accent color
+- [**Separator**](/docs/components/reference#separator) - Add visual spacing and dividers between components
+
+#### New Content Components
+
+- [**Text Display**](/docs/components/reference#text-display) - Add rich markdown-formatted text anywhere in your messages
+- [**Thumbnail**](/docs/components/reference#thumbnail) - An image used in a [section](/docs/components/reference#section)
+- [**Media Gallery**](/docs/components/reference#media-gallery) - Present collections of images in an organized grid
+- [**File**](/docs/components/reference#file) - Embed file attachments as part of your message layout
+
+#### Getting Started
+
+To use the new components, you'll need to send the message flag `1 << 15` (IS_COMPONENTS_V2) which activates the components system on a per-message basis. 
+
+We've created guides to help you implement these new features:
+
+- [Using Message Components](/docs/components/using-message-components) - Learn how to build rich message layouts with components
+- [Using Modal Components](/docs/components/using-modal-components) - Create interactive forms and dialogs
+
+#### Compatibility Notes
+
+[Legacy component behavior](/docs/components/reference#legacy-message-component-behavior) will continue to work as before, so your existing integrations won't break. However, when using the Components V2 flag, you'll need to adapt to a few changes:
+
+- The `content` and `embeds` fields must be sent as components
```

Comment:
The wording here does not sound right. This can be possible misunderstood for users as "there's a component for embeds", which isn't really the case since there's no components to mirror the functionality of embed fields, embed author and embed footer.

Reply:
Agreed, same problem near the top of `docs/components/reference.mdx`